src/engine/routing_algorithms/alternative_path_mld.cpp

```diff
@@ -730,6 +808,9 @@ InternalManyRoutesResult alternativePathSearch(SearchEngineData<Algorithm> &sear
     NodeID shortest_path_via = shortest_path_via_it->node;
     EdgeWeight shortest_path_weight = shortest_path_via_it->weight;
 
+    const double duration_estimation = shortest_path_weight / facade.GetWeightMultiplier();
+    parameters.kAtMostLongerBy = getLongerByFactorBasedOnDuration(duration_estimation);
```

Comment:
Hm, modifying the parameter attribute in-place here is a bit ugly.. but I get why we do it only here.

Reply:
Yeah I wanted to make it explicit what the value is that we really use and not just transform the default in some places.